blocks_colors:  {A: "r", B: "r", C: "b", D: "r", E: "b", F: "r", G: "b", H: "r", I: "b", L: "r", M: "b"}
#block_placed_left:   {A: false, B: false, C: false, D: false, E: false, F: false, G: false, H: false, I: false, L: false, M: false}
#block_placed_right:   {A: false, B: false, C: false, D: false, E: false, F: false, G: false, H: false, I: false, L: false, M: false}

block_grasped:   {A: false, B: false, C: false, D: false, E: false, F: false, G: false, H: false, I: false, L: false, M: false}

blocks_dim: 0.05

table_pos:  [0.7673182, 0.0, 0.3875] # accounting for lower real table
table_dim:  [0.6, 2.0, 0.775]

#MiddlePlacementN: [0.74, 0.0, 0.825]
#Bluebox:          [0.668, 0.608, 0.801]

# here also accounting for lower real table
block_dest_left:  [0.668, 0.608, 0.80] #Bluebox
block_dest_right: [0.74, 0.0, 0.80]    #MiddlePlacementN
